gui/FieldInformation.h

```diff
@@ -37,6 +37,7 @@ struct FieldInformation
 {
   QByteArray name;
   double range[2];
```

Comment:
I wonder if "totalRange" and "displayRange" might be more obvious?

Reply:
"totalRange" makes sense, but the second is not really the display range because the user can display whatever range they like.  It is the automatically estimated range of inlier data using robust estimation.  "autoRange" is probably fine, but "inlierRange" might also work.

Reply:
Okay, makes more sense now, thanks! I guess it's okay either "range" or "totalRange" and either "autoRange" or "inlierRange". Maybe still mild preference for "totalRange" vs. just "range", now that there are multiple 'flavors' of range.

Reply:
...and actually I think I'm also leaning slightly toward "autoRange" now that you've reminded me how it works... :slightly_smiling_face: